join/v0.3/join-v0.3.md

```diff
@@ -0,0 +1,297 @@
+# join v0.3
+
+<h2>for defining *supergraphs* which join multiple *subgraphs*</h2>
+
+```raw html
+<table class=spec-data>
+  <tr><td>Status</td><td>Draft</td>
+  <tr><td>Version</td><td>0.3</td>
+</table>
+<link rel=stylesheet href=https://specs.apollo.dev/apollo-light.css>
+<script type=module async defer src=https://specs.apollo.dev/inject-logo.js></script>
+```
+
+```mermaid diagram -- Schema joining multiple subgraphs
+graph LR
+  classDef bg fill:none,color:#22262E;
+  s1(auth.graphql):::bg-->core(composed schema: photos.graphql)
+  s2(images.graphql):::bg-->core
+  s3(albums.graphql):::bg-->core
+  style core fill:none,stroke:fuchsia,color:fuchsia;
+```
+
+This document defines a number of directives `join` for describing joins between subgraph types in a supergraph.
```

Comment:
This is pasted from v0.2 but it doesn't make sense. I'll open an issue to vet these docs and raise it to the team, no change needed for now.